src/ser/mod.rs

```diff
@@ -97,6 +97,7 @@ impl Default for PrettyConfig {
             new_line: "\r\n".to_string(),
             indentor: "    ".to_string(),
             separate_tuple_members: false,
+            enumerate_arrays: true,
```

Comment:
Hmmmm, are you sure that this option should be enabled by default?

Reply:
Not really. Would you not want to see the indices?

Reply:
I agree with @ozkriff, I think it's too noisy for it to be the default.